src/test/java/org/jilt/test/data/defaultvalue/DefaultValueWithLombok.java

```diff
@@ -0,0 +1,39 @@
+package org.jilt.test.data.defaultvalue;
+
+import org.jilt.Builder;
+
+import java.util.Set;
+
+@Builder
```

Comment:
This PR is missing the functionality of marking a field annotated with `@lombok.Builder.Default` as optional. We would catch it if we made the `@Builder` here `STAGED`.

So, let's change this to:

```suggestion
@Builder(style = BuilderStyle.STAGED)
```

and make sure the `DefaultTest ` test still passes.